src/index.ts

```diff
@@ -595,7 +611,7 @@ function orderedList(textarea: HTMLTextAreaElement): SelectionRange {
       textarea.selectionEnd = endOfLine
     }
   } else {
-    lines = (function() {
+    lines = (function () {
```

Comment:
It could be a good time to make this a named function. This looks like a CoffeeScript remnant.